src/components/contact-components/ContactForm.tsx

```diff
@@ -0,0 +1,167 @@
+  setSuccess,
+  setSubmitted,
+}: Props) => {
+  const formInitState: FormFields = {
+    name: '',
+    email: '',
+    phoneNumber: '',
+    companyName: '',
+    title: '',
+    message: '',
+  }
+
+  const validityInitState: FieldsValidity = {
+    name: false,
+    email: false,
+    phoneNumber: false,
+    companyName: false,
+    title: false,
+    message: false,
+  }
+
+  const t = useTranslations('common.contact.form')
+
+  const [validForm, setValidForm] = useState<boolean>(false)
+  const [formData, setFormData] = useState<FormFields>(formInitState)
+  const [validFields, setValidFields] =
+    useState<FieldsValidity>(validityInitState)
+
+  useEffect(() => {
+    setValidForm(Object.values(validFields).every(Boolean))
+  }, [validFields])
+
+  const handleChange = (
+    event: ChangeEvent<HTMLInputElement | HTMLTextAreaElement>,
+  ) => {
+    const { id, value } = event.target
+    setFormData({ ...formData, [id]: value })
+    setValidFields((validFields) => ({
+      ...validFields,
+      [id]: validateField(id, value),
+    }))
+  }
+
+  const handleSubmit = async () => {
+    try {
+      await sendContactFrom(formData)
+      setSuccess(true)
+      setFormData(formInitState)
+      setValidFields(validityInitState)
+    } catch (error) {
+      setSuccess(false)
+    } finally {
+      setSubmitted(true)
+    }
+  }
+
+  return (
+    <form onSubmit={(e) => e.preventDefault()} className="my-5">
+      <div className="grid grid-cols-1 sm:grid-cols-2">
+        <ContactFormField
+          title={t('name')}
+          type="text"
+          id="name"
+          placeHolder="Példa János"
+          value={formData.name}
+          onChange={(event) => {
+            handleChange(event)
+          }}
+        />
+        <ContactFormField
+          title={t('email')}
+          type="email"
+          id="email"
+          placeHolder="peldaJanos@email.com"
+          value={formData.email}
+          onChange={(event) => {
+            handleChange(event)
+          }}
+        />
+        <ContactFormField
+          title={t('phoneNumber')}
+          type="tel"
+          id="phoneNumber"
+          placeHolder="06012345678"
+          value={formData.phoneNumber}
+          pattern="^(?:\+36|06)?\s?[1-9]\d\s?\d{3}\s?\d{3,4}$|^(?:\+36|06)?\s?((1|20|30|31|50|70|90)\d)\s?\d{3}\s?\d{3,4}$"
+          onChange={(event) => {
+            handleChange(event)
+          }}
+        />
+        <ContactFormField
+          title={t('companyName')}
+          type="text"
+          id="companyName"
+          placeHolder="Kis Kft."
+          value={formData.companyName}
+          onChange={(event) => {
+            handleChange(event)
+          }}
+        />
+        <ContactFormField
+          title={t('title')}
+          type="text"
+          id="title"
+          placeHolder="Osztályvezető"
+          value={formData.title}
+          onChange={(event) => {
+            handleChange(event)
+          }}
+        />
```

Comment:
A titulus legyen a név mellett inkább. Angolban kicsit átverő, hogy a "Title" a szöveged mellett van, így olyan mintha a szövegnek kéne címet adj

Reply:
Én azt úgy értettem, hogy a titulus az a cégen belüli poziciója, és nem egy Dr vagy ifj. Így meg az lenne a logikus ha a cég neve után lenne. Szerintem ha megcsinálom angolul is a placeholder-eket akkor egyértlmű lenne, hogy nem az üzenet címére utal

Reply:
Ezt majd megkérdezem, hogy hogy értették, egyenlöre maradhat így akkor